exchanges/btcmarkets/btcmarkets_wrapper.go

```diff
@@ -380,93 +386,81 @@ func (b *BTCMarkets) ModifyOrder(action *order.Modify) (string, error) {
 func (b *BTCMarkets) CancelAllOrders(_ *order.Cancel) (order.CancelAllResponse, error) {
-	cancelAllOrdersResponse := order.CancelAllResponse{
-		Status: make(map[string]string),
-	}
-	openOrders, err := b.GetOpenOrders()
+	var resp order.CancelAllResponse
+	tempMap := make(map[string]string)
+	orders, err := b.GetOrders("", "", "", "", "")
 	if err != nil {
-		return cancelAllOrdersResponse, err
-	}
-
-	var orderList []int64
-	for i := range openOrders {
-		orderList = append(orderList, openOrders[i].ID)
+		return resp, err
 	}
-
-	if len(orderList) > 0 {
-		orders, err := b.CancelExistingOrder(orderList)
+	for x := range orders {
+		_, err := b.RemoveOrder(orders[x].OrderID)
 		if err != nil {
-			return cancelAllOrdersResponse, err
-		}
-
-		for i := range orders {
-			if !orders[i].Success {
-				cancelAllOrdersResponse.Status[strconv.FormatInt(orders[i].ID, 10)] = orders[i].ErrorMessage
-			}
+			tempMap[orders[x].OrderID] = "Failed"
+		} else {
+			tempMap[orders[x].OrderID] = "Success"
 		}
 	}
-	return cancelAllOrdersResponse, nil
+	return resp, nil
 }
 
 // GetOrderInfo returns information on a current open order
 func (b *BTCMarkets) GetOrderInfo(orderID string) (order.Detail, error) {
-	var OrderDetail order.Detail
-
-	o, err := strconv.ParseInt(orderID, 10, 64)
+	var resp order.Detail
+	o, err := b.FetchOrder(orderID)
 	if err != nil {
-		return OrderDetail, err
-	}
-
-	orders, err := b.GetOrderDetail([]int64{o})
-	if err != nil {
-		return OrderDetail, err
-	}
-
-	if len(orders) > 1 {
-		return OrderDetail, errors.New("too many orders returned")
-	}
-
-	if len(orders) == 0 {
-		return OrderDetail, errors.New("no orders found")
-	}
-
-	for i := range orders {
-		var side order.Side
-		if strings.EqualFold(orders[i].OrderSide, order.Ask.String()) {
-			side = order.Sell
-		} else if strings.EqualFold(orders[i].OrderSide, order.Bid.String()) {
-			side = order.Buy
-		}
-		orderDate := time.Unix(int64(orders[i].CreationTime), 0)
-		orderType := order.Type(strings.ToUpper(orders[i].OrderType))
-
-		OrderDetail.Amount = orders[i].Volume
-		OrderDetail.OrderDate = orderDate
-		OrderDetail.Exchange = b.GetName()
-		OrderDetail.ID = strconv.FormatInt(orders[i].ID, 10)
-		OrderDetail.RemainingAmount = orders[i].OpenVolume
-		OrderDetail.OrderSide = side
-		OrderDetail.OrderType = orderType
-		OrderDetail.Price = orders[i].Price
-		OrderDetail.Status = order.Status(orders[i].Status)
-		OrderDetail.CurrencyPair = currency.NewPairWithDelimiter(orders[i].Instrument,
-			orders[i].Currency,
-			b.GetPairFormat(asset.Spot, false).Delimiter)
-	}
-
-	return OrderDetail, nil
+		return resp, err
+	}
+	resp.Exchange = b.Name
+	resp.ID = orderID
+	resp.CurrencyPair = currency.NewPairFromString(o.MarketID)
+	resp.Price = o.Price
+	resp.OrderDate = o.CreationTime
+	resp.ExecutedAmount = o.Amount - o.OpenAmount
+	resp.OrderSide = order.Bid
+	if o.Side == "Ask" {
+		resp.OrderSide = order.Ask
+	}
+	switch o.Type {
+	case "Limit":
```

Comment:
In line with the other conversion to var request instead of inline strings, can do the same here